### DISCUSSION TITLE
Self-host: How to backup and restore self-hosted version of Affine

### DISCUSSION THREAD

**Discussion by YourDaniel** [toeverything/AFFiNE#7222] [Q&A]
I did posgres dump on my previous installation of Affine.
On another machine I launched a fresh version of Affine and restored my pgsql dump.
I didn't see my previous created notes in new Affine installation.
I tried copying .affine folder, also didn't help.

Is there a proper step-by-step instruction on how to backup and restore Affine?

**Comment by HeJiachen-PM** [comment]
Hi, we are going to release a local-host control pannel in the next major release. Please stay toned.

**Reply by xzessmedia** [reply]
We are definately tuned.. can you be more specific? Eta?

**Reply by TheSpyder** [reply]
Did this ever happen? This is still the top google hit when I look for how to backup and restore.

**Comment by HeJiachen-PM** [comment]
If you actually restored your pg instances with no backup, I think it will be hard to restore your notes. @forehalo maybe a better one to help. Please consider a hosted plan if you are not sure for self-hosted instances.
